scripts/install_third_party.py

```diff
@@ -338,10 +339,56 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def install_redis_cli():
+    """This installs the redis-cli to the local oppia third_party directory so
+    that development servers and backend tests can make use of a local redis
+    cache.
+    """
+    # We need to install redis-cli separately from using manifest.json since it
+    # is a system program and we need to install it after the library is
+    # untarred.
+    try:
+        # Pipe output to /dev/null for silence in console.
+        null = python_utils.open_file('/dev/null', 'w')
+        subprocess.call([
+            './third_party/redis-cli-6.0.6/src/redis-cli', '--version'])
+        null.close()
+        python_utils.PRINT('Redis-cli is already installed.')
+    except OSError:
+        # Redis-cli is not installed, run the script to install it.
+        # NOTE: These should be constants but not sure where to put them since
```

Comment:
You can change this entire comment to:

"We do the installation here (rather than in manifest.json) since we need to use make."